corehq/apps/groups/templates/groups/group_members.html

```diff
@@ -222,9 +222,9 @@
                     <div class="control-group">
                         <label class="control-label" for="group-reporting-input">{% trans "Reporting" %}</label>
                         <div class="controls">
-                            <select class="input-small" name="reporting" id="group-reporting-input" data-value="{{ group.reporting|BOOL }}">
+                            <select class="input-small" name="reporting" id="group-reporting-input">
                                 <option value="true">{% trans "On" %}</option>
```

Comment:
Is there a reason this select and the one above it never default to true but the one in `module_view_careplan.html` does